js/tracker.js

```diff
@@ -3143,7 +3154,7 @@ function addPosition(position) {
             };
 
             marker.shadow = marker_shadow;
-            marker.balloonColor = (vcallsign == "PIE") ? "rpi" : balloon_colors_name[color_index];
+            marker.balloonColor = (vcallsign == "PIE") ? "rpi" : balloon_colors[color_index];
```

Comment:
I'm surprised we still have these special case callsigns in here :/ they don't make sense for sondehub pro